user-data/main.tf

```diff
@@ -48,6 +65,21 @@ EOF
 
     content = "${module.teleport_bootstrap_script.teleport_bootstrap_script}"
   }
+
+  part {
+    content_type = "text/cloud-config"
+
+    content = <<EOF
+#cloud-config
```

Comment:
I don't think you need this comment (based on the existing `cloud-config` parts)